Dockerfile

```
@@ -63,6 +64,8 @@ ENV DJANGO_SETTINGS_MODULE "course_discovery.settings.production"
 
 RUN pip install -r ${DISCOVERY_CODE_DIR}/requirements/production.txt
 
+RUN make pull_translations
```

I'd move this temporary variable here:

```suggestion
RUN make OPENEDX_ATLAS_PULL=true pull_translations
```